controllers/validator.go

```diff
@@ -155,5 +165,10 @@ func (r *DPAReconciler) ValidateVeleroPlugins(log logr.Logger) (bool, error) {
 			}
 		}
 	}
+
+	if foundAWSPlugin && foundLegacyAWSPlugin {
+		return false, errors.New("Only one of aws and legacy-aws DefaultPlugins may be installed")
```

Comment:
As I explain here https://github.com/openshift/oadp-operator/issues/1536, I think we need to improve our error messages

I suggest something like

```go
fmt.Errorf("%s and %s can not be both specified in DPA spec.configuration.velero.defaultPlugins", oadpv1alpha1.DefaultPluginAWS, oadpv1alpha1.DefaultPluginLegacyAWS)
```